lib/book.py

```diff
@@ -1,74 +1,79 @@
 # Book.py
+'''Controls book functions and holds book data'''
 
 import os
-import re
 import json
 import struct
-from glob import glob
-from urllib import urlencode
 from datetime import datetime
 from cStringIO import StringIO
-from shutil import copy, rmtree
+from shutil import copy
 
 from calibre.ebooks.mobi import MobiError
 from calibre.library import current_library_path
 from calibre.ebooks.metadata.mobi import MetadataUpdater
-from calibre.ebooks.metadata.meta import get_metadata, set_metadata
 
 from calibre_plugins.xray_creator.lib.book_parser import BookParser
 from calibre_plugins.xray_creator.lib.book_settings import BookSettings
 from calibre_plugins.xray_creator.lib.xray_db_writer import XRayDBWriter
 from calibre_plugins.xray_creator.lib.goodreads_parser import GoodreadsParser
 
+LIBRARY = current_library_path().replace('/', os.sep)
```

Comment:
What is the reason for this?  Just curious, nothing obviously wrong with doing it.